qa/tasks/cephfs/test_volumes.py

```diff
@@ -662,6 +665,38 @@ def test_volume_info_without_subvolumegroup(self):
                          "'pending_subvolume_deletions' should not be present in absence"
                          " of subvolumegroup")
 
+    def test_volume_info_with_human_readable_flag(self):
+        """
+        Tests the 'fs volume info --human_readable' command
+        """
+        vol_fields = ["pools", "used_size", "pending_subvolume_deletions", "mon_addrs"]
+        group = self._generate_random_group_name()
+        # create subvolumegroup
+        self._fs_cmd("subvolumegroup", "create", self.volname, group)
+        # get volume metadata
+        vol_info = json.loads(self._get_volume_info(self.volname, "--human_readable"))
```

Comment:
Could we verify the value of 'used', 'used_size' and 'avail' ? One way is to verify using the regex of [numbers][B|K|M|G...] based on the display format?

Reply:
As per the `mgr_util.format_bytes()` , it assigns type above `k`.And if our value is very small in some case, or `0` we won't be having any unit assigned instead just a blank space as per the width given.

Reply:
Added checks to verify units.